modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/reducers/languageReducer.es.js

```diff
@@ -105,6 +100,12 @@ const getLocalizedValue = ({
 				return _value;
 			}
 		}
+		case 'numeric':
+		case 'select':
+		case 'text': {
+			return _value;
+		}
+
```

Comment:
did you mean to edit this file?

Reply:
no, I didn't. I think my branch was ahead of yours so these commits were pushed too.